pkgs/applications/editors/emacs/elisp-packages/manual-packages/elisp-ffi/default.nix

```
@@ -28,20 +24,16 @@ in melpaBuild {
   preBuild = ''
     mv ffi.el elisp-ffi.el
     make
- '';
-
-  recipe = writeText "recipe" ''
-   (elisp-ffi :repo "skeeto/elisp-ffi" :fetcher github)
   '';
 
   meta = {
     description = "Emacs Lisp Foreign Function Interface";
     longDescription = ''
-        This library provides an FFI for Emacs Lisp so that Emacs
-        programs can invoke functions in native libraries. It works by
-        driving a subprocess to do the heavy lifting, passing result
-        values on to Emacs.
-      '';
+      This library provides an FFI for Emacs Lisp so that Emacs
+      programs can invoke functions in native libraries. It works by
+      driving a subprocess to do the heavy lifting, passing result
+      values on to Emacs.
+    '';
     license = lib.licenses.publicDomain;
```

```suggestion
    license = lib.licenses.unlicense;
```

Since we have `unlicense` in licenses set, and the package itself says it is unlicense'd.